frontend/apps/reader/readerui.lua

```diff
@@ -270,6 +270,10 @@ function ReaderUI:init()
             ui = self,
             document = self.document,
         })
+        -- Current page
+        self.getCurrentPage = function()
+          return self.paging.current_page
+        end
```

Comment:
For clarity, I think these could just be regular methods: `ReaderUI:getCurrentPage()` with a `if self.document.info.has_pages then` in it (like we have everywhere).
Just so we find it where we expect to find it, instead of having to think about a dynamically plugged function.

(And you call it like a methods, while it really is an unbound function.)

Reply:
done

Reply:
what is the difference between lua methods and functions?

Reply:
```
Object.function(arg1, arg2)
Object:method(arg1, arg2)
```
The `:` implicitely defines self as being expected as the 1st argument, and when you call `instance:method(arg1, arg2)`, it implicitely passes `instance` as `self` in the call.

In the definitions, these are equivalents, but the 1st is more readable:
```
Object:method(arg1, arg2)
Object.function(self, arg1, arg2)
```

Reply:
thanks for that, lua is not my primary language :)